examples/index.html

```diff
@@ -170,7 +170,7 @@ <h1>Share</h1>
                   <div
                     id="map"
                     class="leaflet-map"
-                    style="width: 100% ; height: 300px;"
+                    style="width: 100% ; height: 300px; z-index: 0"
```

Comment:
Adding a colon at the end could save the beginners some headache :)

Reply:
yes. Fixing it now